Source/Microsoft.Teams.Apps.FAQPlusPlus/Bots/FaqPlusPlusBot.cs

```diff
@@ -747,15 +749,22 @@ private static Task<TaskModuleResponse> GetTaskModuleResponseAsync(Attachment qu
                 var botDisplayName = turnContext.Activity.Recipient.Name;
                 var teamWelcomeCardAttachment = WelcomeTeamCard.GetCard();
                 var teamWelcomeFeedbackCardAttachment = WelcomeFeedbackTeamCard.GetCard();
-                if (teamDetails.Team.Name.Contains(Strings.FeedbackCheckOne) || teamDetails.Team.Name.Contains(Strings.FeedbackCheckTwo))
+                /*if (teamDetails.Team.Name.Contains(Strings.FeedbackCheckOne) || teamDetails.Team.Name.Contains(Strings.FeedbackCheckTwo))
                 {
                     await this.SendCardToTeamAsync(turnContext, teamWelcomeFeedbackCardAttachment, teamDetails.Team.Id, cancellationToken).ConfigureAwait(false);
                 }
                 else
+                {
+                    await this.SendCardToTeamAsync(turnContext, teamWelcomeCardAttachment, teamDetails.Team.Id, cancellationToken).ConfigureAwait(false);
+                }*/
+                if (teamDetails.Team.Id == expertID)
                 {
```

Comment:
Add some additional comments here, when I look at this code, I'm not sure if I can really understand what's going on

Reply:
Looks good! I'm going to resolve this conversation accordingly!